certified-connectors/DocuSignDemo/script.csx

```diff
@@ -2976,6 +2976,153 @@ public class Script : ScriptBase
     return body;
   }
 
+  private JObject BulkSendBodyTransformation(JObject body)
+  {
+    var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
+    var name = query.Get("name");
+    JObject newBody = ParseCSV(body);
+    newBody["name"] = name;
+    return newBody;
+  }
+
+  private JObject ParseCSV(JObject inputBody)
+  {
+    var input = inputBody.GetValue("csv").ToString();
+    var body = new JObject();
+
```

Comment:
Please remove extra blank lines through out this method.

Reply:
Sure

Reply:
Done